selinux/Makefile

```diff
@@ -0,0 +1,33 @@
+TARGETS?=hirte
```

Comment:
This is not going to work for RPM installations, we are using meson to build and install, Makefile is available just as a facade for developers and make should call meson to perform stuff (still not true for some invocations like lint or codespell, but we will get there).
So could you please convert it to meson and adapt spec file accordingly?

Reply:
But we will need to exec makefile to build selinux policy, I think.

Reply:
> But we will need to exec makefile to build selinux policy, I think.

If you want I can take a look how to do it in meson

Reply:
The big part that is missing is to integrate this build and install into meson.